### DISCUSSION TITLE
Stopped working after latest chrome update

### DISCUSSION THREAD

**Discussion by AbhaysinghBhosale** [JamesBrill/react-speech-recognition#232] [Q&A]

**Comment by ghost** [comment]
I am having the same issue.
It stopped working on Chrome 122.0.6261.129
Working fine on Edge 124.0.2478.105

**Comment by Mahbub11** [comment]
Same issue Happen on Chrome Version 125.0.6422.60

**Comment by aleksa-codes** [comment]
The same issue here, it is more of a problem related to the Web SpeechRecognition API with Chrome (not Chromium as for me it works as expected on Edge). I also noticed on an Android phone it stopped working properly, one good place to test it is the official Google example: https://www.google.com/chrome/demos/speech.html

Not sure where to report this issue.

**Comment by aleksa-codes** [comment]
If anyone can update me on what is happening or point me to what to do? The Desktop Chrome still transcribes with no punctuation and on the Android Chrome I am getting duplicate interimResults. Everything works as expected on Desktop Edge and on iOS devices.
